README.md

```diff
@@ -32,6 +38,14 @@ git clone git@github.com:joomla-projects/j4adminui.git
 ```bash
 cd j4adminui
 ```
+ ##### PHP LDAP should be active for composer
```

Comment:
these instructions are not correct

Reply:
Good Morning sir 
I'm ravi nandan saxena,
sorry sir,
![Screenshot (92)](https://user-images.githubusercontent.com/44205807/70789299-c1277d00-1db8-11ea-8ac5-eb00b6f2aa35.png)
Please check this screen shot sir, it is clearly stating that php ldap should be active for doing composer install. and i have put the exact instructions in readme ,that helped me activate ldap in windows. then done composer install.

Reply:
1. You don't put instructions for every php module that should be enabled so why this one - just because it was not enabled on your system
2. The instructions for linux are clearly wrong as they are for php5 and joomla 4 requires php7